src/templatizer.js

```diff
@@ -4,46 +4,41 @@ const argumentsTemplate = JSON.parse(fs.readFileSync('./src/template/args.json')
 const optionTemplate = require('./template/option.json');
 
 const templatizer = (context, config) => {
-    argumentsTemplate.definitions.arguments.properties = {};
```

Comment:
I recommend to use pure functions across the project.

Reply:
@xferra we have an issue to fix it  #62

Reply:
It was not clear from the description and seems like you're still missing the main idea: I recommend context-free implementation.